crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -1637,16 +1647,19 @@ pub struct Errors {
 }
 
 #[derive(Default, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Can you confirm TrustPayTransactionStatusErrorResponse also requires camelCase

Reply:
Yes the struct TrustPayTransactionStatusErrorResponse is required to be a camelCase
<img width="978" alt="Screenshot 2023-12-06 at 11 49 36 AM" src="https://github.com/juspay/hyperswitch/assets/131388445/154e71e6-5bdb-4b06-8892-fddf87b289e1">